### PR TITLE
Muzzle Spymemcached

### PR DESCRIPTION
**Pull request by gary-huang**
Add muzzle block and remove class loader matcher method.

### CODE REVIEW

File: dd-java-agent/instrumentation/spymemcached-2.12/spymemcached-2.12.gradle [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
-  versions = "[1.12.0,)"
-  scanMethods = true
-  verifyPresent = [
-    "net.spy.memcached.ConnectionFactoryBuilder": "setListenerExecutorService",
-  ]
+muzzle {
+  pass {
+    group = "net.spy"
+    module = 'spymemcached'
+    versions = "[2.10.0,)"
```
Comment (mar-kolya) [inline dd-java-agent/instrumentation/spymemcached-2.12/spymemcached-2.12.gradle]: Could you please add a comment somewhere onto why 2.12 is used for build/tests and 2.10 is needed?

Comment (gary-huang) [reply dd-java-agent/instrumentation/spymemcached-2.12/spymemcached-2.12.gradle]: Fixed, do you think it's ok?

Comment (mar-kolya) [reply dd-java-agent/instrumentation/spymemcached-2.12/spymemcached-2.12.gradle]: yes, thanks!

Comment (gary-huang) [reply dd-java-agent/instrumentation/spymemcached-2.12/spymemcached-2.12.gradle]: Thank you! :)

**Pull request merged**